### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by tooomm**
## Related Ticket(s)
- Fixes typo from #4293
- Addresses issue mentioned by ebbit in https://github.com/Cockatrice/Cockatrice/pull/4302

## What will change with this Pull Request?
- Use caps on shortcut
- Forgot Password --> Restore Password

### CODE REVIEW

File: cockatrice/src/dlg_forgotpasswordchallenge.cpp [hunk 1]
```diff
@@ -27,9 +27,9 @@ DlgForgotPasswordChallenge::DlgForgotPasswordChallenge(QWidget *parent) : QDialo
     }
 
     if (servers.getFPHostname().isEmpty() && servers.getFPPort().isEmpty() && servers.getFPPlayerName().isEmpty()) {
-        QMessageBox::warning(this, tr("Forgot Password Challenge Warning"),
-                             tr("Oops, looks like something has gone wrong.  Please restart the forgot password "
-                                "process by using the forgot password button on the connection screen."));
+        QMessageBox::warning(this, tr("Reset Password Challenge Issue"),
+                             tr("Oops, looks like something has gone wrong. Please restart the process "
```
Comment (ebbit1q) [inline cockatrice/src/dlg_forgotpasswordchallenge.cpp:31]: I think referencing a "process" might be intimidating to some users that might be less familiar in the language, instead just asking them to try again seems more fitting.

File: cockatrice/src/dlg_forgotpasswordchallenge.cpp [hunk 2]
```diff
@@ -27,9 +27,9 @@ DlgForgotPasswordChallenge::DlgForgotPasswordChallenge(QWidget *parent) : QDialo
     }
 
     if (servers.getFPHostname().isEmpty() && servers.getFPPort().isEmpty() && servers.getFPPlayerName().isEmpty()) {
-        QMessageBox::warning(this, tr("Forgot Password Challenge Warning"),
-                             tr("Oops, looks like something has gone wrong.  Please restart the forgot password "
-                                "process by using the forgot password button on the connection screen."));
+        QMessageBox::warning(this, tr("Reset Password Challenge Issue"),
```
Comment (ebbit1q) [inline cockatrice/src/dlg_forgotpasswordchallenge.cpp:30]: issue is not a very specific word, I preferred warning

File: cockatrice/src/dlg_forgotpasswordchallenge.cpp (hunk 1 above)
Comment (tooomm) [reply cockatrice/src/dlg_forgotpasswordchallenge.cpp:31]: Agreed. Fixing the repetition here is not enough. :)

File: cockatrice/src/dlg_forgotpasswordchallenge.cpp [hunk 3]
```diff
@@ -27,9 +27,8 @@ DlgForgotPasswordChallenge::DlgForgotPasswordChallenge(QWidget *parent) : QDialo
     }
 
     if (servers.getFPHostname().isEmpty() && servers.getFPPort().isEmpty() && servers.getFPPlayerName().isEmpty()) {
-        QMessageBox::warning(this, tr("Forgot Password Challenge Warning"),
-                             tr("Oops, looks like something has gone wrong.  Please restart the forgot password "
-                                "process by using the forgot password button on the connection screen."));
+        QMessageBox::warning(this, tr("Reset Password Challenge Warning"),
+                             tr("A problem has occored. Please try to request a new password again."));
```
Comment (ebbit1q) [inline cockatrice/src/dlg_forgotpasswordchallenge.cpp:31]:
```suggestion
                             tr("A problem has occurred. Please try to request a new password again."));
```

File: cockatrice/src/dlg_forgotpasswordreset.cpp [hunk 4]
```diff
@@ -27,9 +27,8 @@ DlgForgotPasswordReset::DlgForgotPasswordReset(QWidget *parent) : QDialog(parent
     }
 
     if (servers.getFPHostname().isEmpty() && servers.getFPPort().isEmpty() && servers.getFPPlayerName().isEmpty()) {
-        QMessageBox::warning(this, tr("Forgot Password Reset Warning"),
-                             tr("Oops, looks like something has gone wrong. Please re-start the forgot password "
-                                "process by using the forgot password button on the connection screen."));
+        QMessageBox::warning(this, tr("Reset Password Warning"),
+                             tr("A problem has occored. Please try to request a new password again."));
```
Comment (ebbit1q) [inline cockatrice/src/dlg_forgotpasswordreset.cpp:31]:
```suggestion
                             tr("A problem has occurred. Please try to request a new password again."));
```

**Comment by ebbit1q** [comment]
your commit says you found the last one but cockatrice/src/window_main.cpp:1324 still says Forgot Password

**Comment by tooomm** [comment]
Think I got them now. :)

**Review by ebbit1q — APPROVED** [review]

**Pull request merged**